pkgs/applications/networking/instant-messengers/riot/riot-web.nix

```diff
@@ -1,5 +1,6 @@
-{ stdenv, fetchurl, fetchpatch }:
+{ lib, stdenv, fetchurl, fetchpatch, writeText, config ? null }:
```

Comment:
Mhm, `conf` is a pretty generic term that might be set accidentially in `all-packages.nix`

Adding
```
riot-web = callPackage ../applications/networking/instant-messengers/riot/riot-web.nix {
  conf = config.riot-web.conf or null;
};
```

to `all-packages.nix` helps to avoid namespace pollution.
It can be still overridden with the `override` function, but it can be also set in:
`~/.config/nixpkgs/config.nix` via `riot-web.conf`
I think renaming it from `conf` to `config` also don't hurt.

Reply:
I guess you meant renaming from `config` to `conf`? Anyway you're right, I didn't think about the possibility of name conflict, changes applied, thank you :)